### ISSUE TITLE
Privacy protection 

### ISSUE THREAD

**Issue by Discostu36** [languagetool-org/languagetool#3674]
Privacy protection on for LT Plus needs to be worked on to comply to current EU law:

- User data can't be transferred to fastspring under EU Privacy Shield anymore, as Privacy Shield has been declared as illegal by European Court of Justice. As far as I know you need to use standard contractual clauses for that now.
- Usage of Google Analytics on languagetool.org needs Opt-In by users to comply with GDPR

Disclaimer: I am not a lawyer, this is just the knowledge I obtained by working on my employer's website

**Comment by grahamperrin** [comment]
Also #6186

**Comment by markcellus** [comment]
@Discostu36 did you ever getany answers to your original question? Maybe perhaps outside of this issue? I'm starting to see AI-generated prompts in the use of LanguageTool (see screenshot below) and if that's being generated by a third-party tool like ChatGPT, etc seems like that should also be opt-in as its likely collecting data there as well.

![Selection_012](https://github.com/languagetool-org/languagetool/assets/1446916/352e0a95-d711-4947-87a0-491f19f9dbda)

**Comment by Discostu36** [comment]
No, I didn't receive any feedback elsewhere. I also don't know if the AI generated feedback comes from LT's own AI or a third party.
